src/signals/incident-management/containers/Dashboard/components/BarChart/vega-config-bar-chart.ts

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+
+export const vegaConfigBarChart: any = {
+  $schema: 'https://vega.github.io/schema/vega-lite/v5.json',
+  description: 'A bar chart showing cases with their statusses',
+  data: {
+    values: [
+      { a: 'Heropend', b: '18' },
```

Comment:
You can change field "a" and "b" to something more descriptive.

Reply:
These will be overwritten by the fetch data, so I see no need.